.github/workflows/ci.yml

```diff
@@ -76,6 +76,15 @@ jobs:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/ # Needed for auth
 
+      # Setup caching to optimize task execution across runs.
+      # Note: Wireit documentation uses release tags, like "google/wireit@setup-github-actions-caching/v2.0".
+      #       However, Backstage uses commit hashes to ensure reproducibility.
+      #       Because Wireit uses a repo branch for their GitHub Actions code,
+      #       1. Find the appropriate release tag at https://github.com/google/wireit/tags
+      #       2. Click on the commit hash for the release tag e.g. "eea3c9f".
+      #       3. Use the full commit hash in the `uses` below.
```

Comment:
We use hashes to harden the workflows and reduce the risk of supply chain attacks. We use Renovate to bump them, so it would be important that it behaves correctly with these actions and doesn't bump across different tag sets.